sdk/keyvault/keyvault-keys/review/keyvault-keys.api.md

```diff
@@ -310,7 +349,7 @@ export interface KeyVaultKeyIdentifier {
 export type KeyWrapAlgorithm = "A128KW" | "A192KW" | "A256KW" | "RSA-OAEP" | "RSA-OAEP-256" | "RSA1_5";
 
 // @public
-export const enum KnownDeletionRecoveryLevel {
+export enum KnownDeletionRecoveryLevel {
```

Comment:
These are the result of regenerating after https://github.com/Azure/autorest.typescript/issues/1013

Reply:
Why do these need to be declared, though? They are read-only properties returned from the service and could be any string. IIRC, the other languages don't define enums for them and just declare the property as a `string`.

Reply:
It's how TypeScript declares and models extensible enums. The type of `DeletionRecoveryLevel` is a string and we export a set of `known` values that can be used interchangeably with it. 

There are reasons why we need to go this way (actual field's type is a string and we also expose a Known<xyz> enums that can be used instead of the raw string) with enums in TypeScript 

With _all that said_ looking at DeletionRecoveryLevel specifically I don't see anywhere it's used as input so I agree these should not exist 😄 why they were added originally, I don't know (was before my time, and looks like the original union type was introduced in 4.1 so there's some history there) but I'll create an issue for us to audit and deprecate these where they are only output values...

Reply:
We probably don't need to deprecate them, but we should probably avoid adding new ones for read-only model properties.